.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
```

Comment:
Have you missed PR's or do we just want to do it issues first to test?

Reply:
I was just going to to Issues first since we have a relatively good handle on PRs.

Reply:
In which case (I know I know, I'm a stickler....)

```suggestion
name: Mark stale issues
```